script.js

```diff
@@ -1,8 +1,108 @@
-const headerBurger = document.querySelector('.header-burger');
-const headerMenu = document.querySelector('.header-menu');
-const body = document.querySelector('body');
-headerBurger.onclick = function(){
-  headerBurger.classList.toggle('active');
-  headerMenu.classList.toggle('active');
-  body.classList.toggle('lock');
+'use strict'
+
+const isMobile = {
+  Android: function () {
+    return navigator.userAgent.match(/Android/i);
+  },
+  BlackBerry: function () {
+    return navigator.userAgent.match(/BlackBerry/i);
+  },
+  IOS: function () {
+    return navigator.userAgent.match(/Iphone|iPad|iPod/i);
+  },
+  Opera: function () {
+    return navigator.userAgent.match(/Opera Mini/i);
+  },
+  Windows: function () {
+    return navigator.userAgent.match(/IEMobile/i);
+  },
+  any: function () {
+    return (
+      isMobile.Android() ||
+      isMobile.BlackBerry() ||
+      isMobile.IOS() ||
+      isMobile.Opera() ||
+      isMobile.Windows()
+    );
+  }
 }
+if(isMobile.any()){
+  document.body.classList.add('_touch');
+  let menuArrows = document.querySelectorAll('.menu-arrow');
+  if(menuArrows.length > 0){
+    for(let i = 0; i < menuArrows.length; i++){
+      const menuArrow = menuArrows[i];
+      menuArrow.addEventListener('click', function(e){
+        menuArrow.parentElement.classList.toggle('_active')
+      });
+    }
+  }
+}else{
+  document.body.classList.add('_pc')
+}
+const iconMenu = document.querySelector('.header-burger');
+if(iconMenu){
+  const menuBody = document.querySelector('.menu-body');
+  iconMenu.addEventListener('click', function (e) {
+    document.body.classList.toggle('_lock')
+    iconMenu.classList.toggle('_active');
+    menuBody.classList.toggle('_active')
+  });
+}
+
+document.querySelector('.proj-input').oninput = function(){
+  setTimeout(() => {
```

Comment:
This whole fucntion is a huge mess. I'm sure there's a much cleaner way to do it.